src/components/PlayerComponent.react.js

```diff
@@ -0,0 +1,214 @@
+     * null, it doesn't do anything.
+     */
+    setSeekTo(){
+        const {
+            seekTo,
+            setProps
+        } = this.props;
+
+        if (seekTo !== null && setProps !== null){
+            this.refs.player.seekTo(seekTo);
+            setProps({seekTo: null});
+        }
+    }
+
+    componentDidUpdate(){
+        this.setSeekTo();
+    }
+
+    render() {
+        const {
+            url,
+            playing,
+            loop,
+            controls,
+            volume,
+            muted,
+            playbackRate,
+            width,
+            height,
+            style,
+            progressInterval,
+            playsinline
+        } = this.props;
+
+        return (
+            <ReactPlayer
+                ref="player"
+                url={url}
+                playing={playing}
+                loop={loop}
+                controls={controls}
+                volume={volume}
+                muted={muted}
+                playbackRate={playbackRate}
+                width={width}
+                height={height}
+                style={style}
+                progressInterval={progressInterval}
+                playsline={playsinline}
+            />
+
+        );
+    }
+}
+
+PlayerComponent.propTypes = {
+    /**
+     * The ID used to identify this compnent in Dash callbacks
+     */
+    id: PropTypes.string,
+
+    /**
+     * Dash-assigned callback that should be called whenever any of the
+     * properties change
+     */
+    setProps: PropTypes.func,
+
+    /**
+     * The url of a video or song to play
+     * ◦  Can be an array or MediaStream object
+     */
+    url: PropTypes.string,
+
+    /**
+     * Set to true or false to pause or play the media
+     */
+    playing: PropTypes.bool,
+
+    /**
+     * Set to true or false to loop the media
+     */
+    loop: PropTypes.bool,
+
+    /**
+     * Set to true or false to display native player controls
+     * Vimeo, Twitch and Wistia player will always display controls
+     */
+    controls: PropTypes.bool,
+
+    /**
+     * Set the volume of the player, between 0 and 1
+     * null uses default volume on all players
+     */
+    volume: PropTypes.number,
+
+    /**
+     * Mutes the player
+     * Only works if volume is set
+     */
+    muted: PropTypes.bool,
+
+    /**
+     * Set the playback rate of the player
+     * Only supported by YouTube, Wistia, and file paths
+     */
+    playbackRate: PropTypes.number,
+
+    /**
+     * Set the width of the player
+     */
+    width: PropTypes.string,
+
+    /**
+     * Set the height of the player
+     */
+    height: PropTypes.string,
+
+    /**
+     * Add inline styles to the root element
+     */
+    style: PropTypes.object,
+
+    /**
+     * The time between onProgress callbacks, in milliseconds
+     */
+    progressInterval: PropTypes.string,
+
+    /**
+     * Applies the playsinline attribute where supported
```

Comment:
what does this attribute do?

Reply:
It gives the option to play the video inline. E.g. https://webkit.org/blog/6784/new-video-policies-for-ios/. I improved the docstring to be more descriptive.